backend/src/main/resources/schema.graphqls

```diff
@@ -203,11 +203,11 @@ type Mutation {
     facilityId: String!
     testingFacilityName: String!
     cliaNumber: String
-    street: String
+    street: String!
     streetTwo: String
     city: String
-    county: String
-    state: String
+    county: String @deprecated(reason: "county is derived from smarty streets lookup")
```

Comment:
Nice. I feel like I should probably start attaching that to some elements of DeviceType sometime soon...